packages/hotspot-utils/src/mplAssetAPI.ts

```diff
@@ -124,20 +124,19 @@ export async function getAssetProof(
   }
 }
 
-
 export type AssetsByOwnerOpts = {
-  sortBy?: any;
+  sortBy?: { sortBy: "created"; sortDirection: "asc" | "desc" };
   limit?: number;
   page?: number;
   before?: string;
   after?: string;
-}
+};
 
 export async function getAssetsByOwner(
   url: string,
   wallet: string,
   {
-    sortBy = "created",
```

Comment:
`"Created"` wasn't working. I chatted with Jon Wong from metaplex, turns out this is now an object.